### PR TITLE
Add 'the rush age' blog post

### PR DESCRIPTION
**Pull request by srodrigo**

### CODE REVIEW

**Comment by alvarogarcia7** [comment]
Right before the list of steps, I think a link to this fits well: http://www.nytimes.com/2016/06/12/fashion/internet-technology-phones-introspection.html?_r=0

**Comment by alvarogarcia7** [comment]
Think ", really appreciated in certain kind of devices" is noise

**Comment by alvarogarcia7** [comment]
Maybe when "I'm just concerned that the amount of pressure (sometimes self-imposed)", add a link to Sandro's "the wrong notion of time": https://www.javacodegeeks.com/2012/12/the-wrong-notion-of-time.html

**Comment by alvarogarcia7** [comment]
Interesting article for reflection and food for thought 👍

File: _posts/2016-07-04-the-rush-age.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      href: http://www.photo-street.com/2011/11/always-in-rush.html
+tags:
+- craftsmanship
+- practices
+alias: [/2016/07/04/The-Rush-Age]
+---
+
+We live in the rush age. We are literally overwhelmed by the amount of things we have to do, both at work and outside it. We leave work and have a bunch of emails, Facebook notifications, tweets, text messages (among others), waiting for response. Even worse, we are in bed keeping an eye open looking at our mobile phone in case we get a new notification. This is ridiculous.
+
+As software developers, the same thing happens at work. The feeling is that we need to be more and more productive. We have access to a **huge** amount of information and tools in order to achieve this (Google, Stack Overflow, that new library that claims to simplify your life 1% better than the one your are currently using, etc.). But the feeling I get is that all of this is making us consuming machines. We consume information, process it and pack it again as soon as possible. Our output? Mostly code. And how does this work? Let's see an example:
```
Comment (neomatrix369) [inline _posts/2016-07-04-the-rush-age.md]: making us consuming machines ==> making us like "consuming machines"  -- do you want to say this?

File: _posts/2016-07-04-the-rush-age.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+So we are coding, and we get stuck. Here is a common algorithm:
+
+1. Look at the time tracking tool and see the task should already be finished; we are late. We feel the icy breath of the time on our nape.
```
Comment (neomatrix369) [inline _posts/2016-07-04-the-rush-age.md]: We feel the icy breath of the time on our nape. ==> We feel the icy breath of time on our nape.

File: _posts/2016-07-04-the-rush-age.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+2. Breathe.
+3. Go straight away to Google and type (or copy & paste) the error or question.
+4. Stack Overflow comes to the rescue. Our adrenaline gets out of control before clicking on the first link.
+5. Skip the question without even reading it and scroll down to the answers. There is that glorious green tick for the accepted one. Problem solved!
+6. Breathe again.
+7. Copy the solution and paste it in your code. Quickly. Seconds matter!
+8. If the solution doesn’t work out of the box, tweak it or repeat from 2.
+
+Has anyone experienced this? I did, and I find it terrifying. We are in such a rush that can even fall into the trap of stopping thinking and blindly apply pre-baked solutions.
```
Comment (neomatrix369) [inline _posts/2016-07-04-the-rush-age.md]: stopping thinking ==> stopping to think

File: _posts/2016-07-04-the-rush-age.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+The same thing applies for our operating system. I've used Mac OSX, Windows and Linux (specially this one). Lately, I've been using Mac both at work and at home. It makes you more productive, no doubt about that (I think they balance stability, user interface and access to low level - alias 'console'). But that simplicity makes you lazier in the long term and therefore you lose the ability to **think and solve problems**. Try to go back to a Linux distribution that needs some configuration, after spending 3-4 years on Mac/Windows. You think that you don’t need this? Then you have to set up a custom server for production at work. You will be surprised by how tough it is, not because Linux is that tough (ok, this is arguable), but because it requires you to **understand** what you are doing. Does this example look similar to over-using Google and libraries or frameworks without understanding what they do, or to having troubles with writing our own solution in case we need to? It does to me, and it leads to the same problems.
```
Comment (neomatrix369) [inline _posts/2016-07-04-the-rush-age.md]: long term  ==> long run

Comment (neomatrix369) [inline _posts/2016-07-04-the-rush-age.md:41]: long term ==> long run

**Comment by neomatrix369** [comment]
@srodrigo good post 👍 - hope your okay with my comments

File: _posts/2016-07-04-the-rush-age.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+date: 2016-07-04 10:00:00 +00:00
+author: Sergio Rodrigo Royo
+canonical:
+    name: my personal blog
+    href: http://srodrigo.me/?p=183
+image:
+    src: /assets/img/custom/blog/2016-07-04-the-rush-age/always-in-a-rush.jpg
+    attribution:
+      text: by Mayank Pandey
+      href: http://www.photo-street.com/2011/11/always-in-rush.html
```
Comment (dhatanian) [inline _posts/2016-07-04-the-rush-age.md]: Do we have the rights for this image? I can't find any licensing info on the link

File: _posts/2016-07-04-the-rush-age.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+tags:
+- craftsmanship
+- practices
+alias: [/2016/07/04/The-Rush-Age]
+---
+
+We live in the rush age. We are literally overwhelmed by the amount of things we have to do, both at work and outside it. We leave work and have a bunch of emails, Facebook notifications, tweets, text messages (among others), waiting for response. Even worse, we are in bed keeping an eye open looking at our mobile phone in case we get a new notification. This is ridiculous.
```
Comment (dhatanian) [inline _posts/2016-07-04-the-rush-age.md]: waiting for a response / waiting for our response

File: _posts/2016-07-04-the-rush-age.md (hunk 1 above)
Comment (dhatanian) [inline _posts/2016-07-04-the-rush-age.md]: I'd say 

> turning us into consuming machines

File: _posts/2016-07-04-the-rush-age.md (hunk 2 above)
Comment (dhatanian) [inline _posts/2016-07-04-the-rush-age.md]: I think the expression would be more "down my neck" as in:

> We feel the icy breath of time down our neck

File: _posts/2016-07-04-the-rush-age.md (hunk 5 above)
Comment (srodrigo) [inline _posts/2016-07-04-the-rush-age.md]: Good point. I struggled to find an image that didn't have any issues. I've seen that most people use images from flickr, I'll try to find one from there.

**Pull request merged**